### PR TITLE
Add a DaemonSet table printer

### PR DESCRIPTION
**Pull request by smarterclayton**
Only the last commit is new

### CODE REVIEW

**Comment by dims** [comment]
@smarterclayton looks like verify-bazel.sh is failing

**Comment by smarterclayton** [comment]
@kubernetes/sig-cli-pr-reviews

**Comment by fabianofranz** [comment]
@smarterclayton rebase?

File: pkg/printers/internalversion/printers.go [hunk 1]
```diff
@@ -163,8 +161,21 @@ func AddHandlers(h printers.PrintHandler) {
 	h.TableHandler(replicaSetColumnDefinitions, printReplicaSet)
 	h.TableHandler(replicaSetColumnDefinitions, printReplicaSetList)
 
-	h.Handler(daemonSetColumns, daemonSetWideColumns, printDaemonSet)
-	h.Handler(daemonSetColumns, daemonSetWideColumns, printDaemonSetList)
+	daemenSetColumnDefinitions := []metav1alpha1.TableColumnDefinition{
```
Comment (fabianofranz) [inline pkg/printers/internalversion/printers.go]: `daemon`

**Pull request merged**